src/UsgsAstroFrameSensorModel.cpp

```diff
@@ -147,9 +152,17 @@ csm::ImageCoordCovar UsgsAstroFrameSensorModel::groundToImage(const csm::EcefCoo
                                    double *achievedPrecision,
                                    csm::WarningList *warnings) const {
 
-    throw csm::Error(csm::Error::UNSUPPORTED_FUNCTION,
-      "Unsupported function",
-      "UsgsAstroFrameSensorModel::groundToImage");
+    csm::EcefCoord gp;
+    gp.x = groundPt.x;
+    gp.y = groundPt.y;
+    gp.z = groundPt.z;
+
+    csm::ImageCoord ip = groundToImage(                                          
+      gp, desiredPrecision, achievedPrecision, warnings);                     
+   csm::ImageCoordCovar result(ip.line, ip.samp);                                    
+   // This is a partial, incorrect implementation to test if SocetGXP needs
+   // this method implemented in order to load the sensor.
+   return result;
```

Comment:
This is important that this is not a full implementation, just a test for GXP compliance.